src/Access.hs

```diff
@@ -190,6 +190,7 @@ isStudent u = isJust $ find (has _Student) (u ^.. userRoles)
 
 delegationCapabilities :: User -> Maybe User -> Maybe IdeaSpace -> Maybe Phase -> [Capability]
 -- guards: no delegation in these situations
+delegationCapabilities _ _ _ (Just PhaseVoting{})              = []
```

Comment:
Ok, in this case, the original issue has no sense. As the current implementation forbids to delegate your vote only in result phase. The issue reported the current behavior as the desired one. Probably the usage of godadmin produced this.